bot/converters.py

```diff
@@ -281,22 +281,50 @@ class ISODateTime(Converter):
 
 
 class FetchedUser(Converter):
-    """Fetches from the Discord API and returns a `discord.User` object."""
+    """
+    Fetches from the Discord API and returns a `discord.User` or `discord.Object` object, given an ID.
+
+    If the fetching is successful, a `discord.User` object is returned. If it fails and
+    the error doesn't imply the user doesn't exist, then a `discord.Object` is returned
+    via the `user_proxy` function.
+    """
+
+    # XXX: `proxy_user` shouldn't be here as a helper.
+    #  Should wait for PR #651 to import from bot.utils.whatever, maybe?
+    @staticmethod
+    def proxy_user(user_id: str) -> discord.Object:
+        """
+        Create a proxy user object from the given id.
+
+        Used when a Member or User object cannot be resolved.
+        """
+        log.trace(f"Attempting to create a proxy user for the user id {user_id}.")
+
+        try:
+            user_id = int(user_id)
+        except ValueError:
+            raise BadArgument
+
+        user = discord.Object(user_id)
+        user.mention = user.id
+        user.avatar_url_as = lambda static_format: None
+
+        return user
```

Comment:
No much effort here because this won't stay.

In my opinion, maybe we should wait for #651's `ProxyUser` class in `utils` (but I can't tell if there would be another issue importing from there, correct me if I'm wrong - I should have tested it.)